tasks/README.md

```diff
@@ -262,6 +262,9 @@ Param names will be prefixed with `PT_` to create the environment variables in t
 
 String parameters will not be quoted. Numerical parameters and structured objects will use their JSON representations.
 
+### Positional 
+Positional input is useful for converting existing scripts to tasks.  The parameters are passed to the task as arguments.  The positional/order they are passed should be defined by the parameter.   If the position has not been specified then each parameter will be passed to the task in a unspecified order.
```

Comment:
The `positional` property should be added under the `Task Paremeters` section. We should also mark this as part of `rev 4` of the spec.